pulsar-client-go/pulsar/consumer.go

```diff
@@ -44,6 +44,10 @@ const (
 	// Multiple consumer will be able to use the same subscription name but only 1 consumer will receive the messages.
 	// If that consumer disconnects, one of the other connected consumers will start receiving messages.
 	Failover
+
+	// Multiple consumer will be able to use the same subscription and all messages with the same key
+	// will be dispatched to only one consumer
+	keyShared
```

Comment:
This should be start with capital letter too, like `Exclusive` and `Failover`

Reply:
yes, should is `KeyShared`